drivers/SmartThings/zigbee-switch/fingerprints.yml

```diff
@@ -140,7 +140,22 @@ zigbeeManufacturer:
     deviceLabel: Aqara Smart Wall Switch H1 EU (No Neutral, Double Rocker) 1
     manufacturer: LUMI
     model: lumi.switch.l2aeu1
-    deviceProfileName: aqara-switch-no-power    
+    deviceProfileName: aqara-switch-no-power
+  - id: "LUMI/lumi.switch.b1nacn01"
+    deviceLabel: Aqara Smart Wall Switch T1 (With Neutral, Single Rocker)
+    manufacturer: LUMI
+    model: lumi.switch.b1nacn01
+    deviceProfileName: aqara-switch-power
+  - id: "LUMI/lumi.switch.b2nacn01"
+    deviceLabel: Aqara Smart Wall Switch T1 (With Neutral, Double Rocker) 1
```

Comment:
Can you remind me why the `1` is needed on the end of the device label?

Reply:
https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1384#pullrequestreview-2111510458

Please refer here